sis_scraper/main.py

```diff
@@ -282,6 +282,7 @@ def toTitle(text):
         "R": 3 * 16 * 6,
         "F": 4 * 16 * 6,
         "S": 5 * 16 * 6,
+        "U": 6 * 16 * 6,
```

Comment:
@benjaminrsherman I'm not sure what the max bit length I can stretch things out to, let me know if this needs additional changes

Reply:
It was already at the max, so you'd need to add another bit field in the rust.  Why does RPI need sunday classes :(

Reply:
With 10 64-bit slots, we can't evenly fit the same granularity across all 7 days.  We can either choose to have 10-minute segments from 7:00-22:00 each day, or we can have something more complex which has a higher granularity/longer range on weekdays compared to weekends.  I think it'll be fine to just have 7:00-22:00 since it's really unlikely that classes will be wholly outside that range, so conflict checking would still "work" with them - you'd just have to include a check in the scraper to cap out-of-bound times into that range.  @elihschiff what do you think?

Reply:
@johnnyapol I've updated `sis_scraper.py` to make this change easier - you should be able to just edit the constants in the file now.

Reply:
I've rebased on top of the new changes ... at the moment I haven't changed it to 22:00 from 23:00 so we're at 11 uint64's instead of 10, should I proceed with that?

Reply:
I'd put it at 22:00 for now - if it turns out that the new schedule is _really_ screwy with classes outside of that range we can revisit it, but I'd rather have the improved performance if we can afford it.